modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/AbstractComponentDefinitionParser.java

```diff
@@ -81,6 +80,7 @@ protected Builder doParse(Builder definitionBuilder)
                                        fromSimpleParameter(TARGET_VALUE_PARAMETER_NAME).build())
         .withSetterParameterDefinition(CURSOR_PROVIDER_FACTORY_FIELD_NAME,
                                        fromChildConfiguration(CursorProviderFactory.class).build())
+        .withSetterParameterDefinition("errorMappings", fromChildCollectionConfiguration(EnrichedErrorMapping.class).build())
```

Comment:
this part I still don't underestand... so nobody was paying attention to this in 4.3?

Reply:
it was set by custom code in the parser, passing it through the annotations of the componet :ugly: